Team417_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team417_CENTERSTAGE/competitionprograms/Config.java

```diff
@@ -0,0 +1,201 @@
+
+        HashMap<Boolean, String> closeSummaryStrings = new HashMap<>();
+        closeSummaryStrings.put(true, "\u23f1\ufe0f CLOSE");
+        closeSummaryStrings.put(false, "\u231b FAR");
+        summaryStrings.add(closeSummaryStrings);
+
+        HashMap<Boolean, String> openCVSummaryStrings = new HashMap<>();
+        openCVSummaryStrings.put(true, "\uD83D\uDCF8 OPENCV");
+        openCVSummaryStrings.put(false, "\uD83D\uDCA5 DISTANCE");
+        summaryStrings.add(openCVSummaryStrings);
+
+        HashMap<Boolean, String> backdropSummaryStrings = new HashMap<>();
+        backdropSummaryStrings.put(true, "\uD83D\uDFE1 BACKDROP YES");
+        backdropSummaryStrings.put(false, "\u26d4 BACKDROP NO");
+        summaryStrings.add(backdropSummaryStrings);
+
+        HashMap<Boolean, String> driveToSummaryStrings = new HashMap<>();
+        driveToSummaryStrings.put(true, "\uD83D\uDE97 DRIVE TO YES");
+        driveToSummaryStrings.put(false, "\uD83C\uDFC3 DRIVE TO NO");
+        summaryStrings.add(driveToSummaryStrings);
+
+        HashMap<Boolean, String> aprilTagSummaryStrings = new HashMap<>();
+        aprilTagSummaryStrings.put(true, "\uD83C\uDFFD APRIL TAGS YES");
+        aprilTagSummaryStrings.put(false, "\u274c APRIL TAGS NO");
+        summaryStrings.add(aprilTagSummaryStrings);
+
+        defaults = new ArrayList<>();
+        defaults.add(isRed);
+        defaults.add(isClose);
+        defaults.add(useOpenCV);
+        defaults.add(doBackdropPixel);
+        defaults.add(useDriveTo);
+        defaults.add(useAprilTags);
+    }
+
+    static int choice;
+    static boolean aIsPressed;
+    static boolean xIsPressed;
+    static boolean upIsPressed;
+    static boolean downIsPressed;
+
+    /**
+     * Run the menu. The resulting state can be found in the public fields of this class.
+     */
+    static boolean menu(Telemetry telemetry, Gamepad gamepad) {
+        boolean end = false;
+
+        telemetry.addLine("We are playing on the ");
+        telemetry.addLine(String.format("%s%s alliance, positioned ",
+                choice == 0 ? "->" : "  ", configStrings.get(0).get(isRed)));
+        telemetry.addLine(String.format("%s%s the backdrop, using ",
+                choice == 1 ? "->" : "  ", configStrings.get(1).get(isClose)));
+        telemetry.addLine(String.format("%s%s as our Team Prop Detection, ",
+                choice == 2 ? "->" : "  ", configStrings.get(2).get(useOpenCV)));
+        telemetry.addLine(String.format("%s%s the yellow pixel on the backdrop",
+                choice == 3 ? "->" : "  ", configStrings.get(3).get(doBackdropPixel)));
+        telemetry.addLine(String.format("%s%s drive assistance to the backboard, and ",
+                choice == 4 ? "->" : "  ", configStrings.get(4).get(useDriveTo)));
+        telemetry.addLine(String.format("%s%s information from April Tags into our pose estimate (ONLY IN TELEOP).",
+                choice == 5 ? "->" : "  ", configStrings.get(5).get(useAprilTags)));
```

Comment:
Why do these all have "magic" constants (i.e., "4" magically refers to `useDriveTo`)? At the minimum, named Enums should be used. The names can replace all magic constants referenced here.

As well, I don't see any benefit here from having all of the configStrings in an array. Why not simply have them a named individual variables?